lib/src/vector_drawable.dart

```diff
@@ -23,6 +23,8 @@ final Paint _grayscaleDstInPaint = Paint()
 /// Base interface for vector drawing.
 @immutable
 abstract class Drawable {
+  String get id;
```

Comment:
Please add a docstring.

Probably something like

> A string that should uniquely identify this [Drawable] within its [DrawableRoot].

Is fine.

Reply:
Just added document as you suggested, thank you